pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -405,6 +411,11 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	// we kill at most a single pod during each eviction interval
```

Comment:
This comment should be updated, I guess.

Reply:
Well, it's still true, each loop of this function will only evict one pod. It's just that the loop may return quicker than a pod takes to terminate